api/pkg/apis/v1alpha1/managers/stage/stage-manager.go

```diff
@@ -203,10 +208,19 @@ func (s *StageManager) ResumeStage(ctx context.Context, status model.StageStatus
 				nextStage := ""
 				if currentStage, ok := cam.Stages[stage]; ok {
 					parser := utils.NewParser(currentStage.StageSelector)
-
 					eCtx := s.VendorContext.EvaluationContext.Clone()
 					eCtx.Context = ctx
 					eCtx.Namespace = namespace
+					activationState, err := s.apiClient.GetActivation(
```

Comment:
Now I start to worry about remote stage execution, which is a key scenario in distributed workflow. Originally, the idea is the trigger data contains everything that is needed to carry out a stage. We need to make sure remote stages still work.

Reply:
Related issue #525 for tracking the pending tests for multisite.

Reply:
Will complete this PR at first to unblock the public preview release. Why: this PR brings a new behavior for inputs in activation. We should fix before public preview, otherwise it will bring breaking change. 

Since public preview release doesn't have multi-site scenarios, we can test after public preview release.